app/presenters/instance_presenter.rb

```diff
@@ -15,7 +15,7 @@ def contact_account
   end
 
   def user_count
-    Rails.cache.fetch('user_count') { User.count }
+    Rails.cache.fetch('user_count') { User.confirmed.count }
```

Comment:
Putting aside whether this is a good idea to change or not, this specific way of changing it is going to lead to mixed results ... because the cached result is set elsewhere and is going to still be the total user count. Only in cases where there is a cache miss is this code going to be run.